src/classes/SurrealDB.php

```diff
@@ -0,0 +1,134 @@
+<?php 
+namespace SurrealDB\classes;
+
+require __DIR__."/../interfaces/ISurrealDB.php";
+require __DIR__."/WebService.php";
+
+use SurrealDB\interfaces\ISurrealDB;
+use SurrealDB\WebService\WebService;
+class  SurrealDB implements ISurrealDB
+{
+    private $hostname;
+    private $port;
+    private $username;
+    private $password;
+    private $db;
+    private $ns;
+    private $isSSH;
+    
+    public function getHostname()
+    {
+        return $this->hostname;
+    }
+
+    public function getPort()
+    {
+        return $this->port;
+    }
+
+    public function getUsername()
+    {
+        return $this->username;
+    }
+
+    public function getPassword()
+    {
+        return $this->password;
+    }
+
+    public function getDb()
+    {
+        return $this->db;
+    }
+
+    public function getNs()
+    {
+        return $this->ns;
+    }
+    
+    public function getIsSSH()
```

Comment:
I think you mean `getIsTLS`? This is for encryption, right? https

Reply:
Yes, it's for encryption, because you may have it up locally on http or on a server without encryption.